package.json

```diff
@@ -74,11 +74,6 @@
     "tslint-config-prettier": "^1.18.0",
     "typescript": "~4.7.3"
   },
-  "overrides": {
-    "@yarnpkg/core": {
-      "cross-spawn": "^7.0.5"
```

Comment:
this is no longer needed, because `snyk-nodejs-lockfile-parser` pulls in a newer `cross-spawn` package.

This change also drops support for node <18